public/the_bridge_templates/marketplace/index.html

```diff
@@ -59,11 +59,21 @@ <h3 class="title">
                 DETAILS
               </a>
               -->
-              <a ng-href="#" class="details-link">
+              <a ng-click="changeVote('up', product.id)" class="details-link up_{{product.id}}" style="margin-right:40px;">
                 +<i class="fa fa-cube"></i>
               </a>
+
+              <a ng-click="changeVote('down', product.id)" class="details-link down_{{product.id}}">
```

Comment:
The spec was to have a 'like' style mechanic, where 1 icon could be either 'voted' or 'not voted'.

There appears to be a issue in the logic, in that a user could up vote something, refresh and upvote again.